### PR TITLE
Add SIG Docs Spotlight

### PR DESCRIPTION
**Pull request by PurneswarPrasad**
SIG Docs Spotlight blog added, done within the context of Contribex-comms
cc @sftim

### CODE REVIEW

**Comment by sftim** [comment]
This will mirror https://github.com/kubernetes/contributor-site/pull/323

File: content/en/blog/_posts/2022-07-29-SIG-Docs-Spotlight/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Spotlight on SIG Docs"
+date: 2022-07-29
+slug: sig-docs-spotlight-2022
+---
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-29-SIG-Docs-Spotlight/index.md:6]:
```suggestion
canonicalUrl: https://kubernetes.dev/blog/2022/07/29/sig-docs-spotlight-2022/
---
```
This marks the original as the official source of the article.

**Review by sftim — COMMENTED** [review]
/lgtm
/approve
/hold

**Comment by sftim** [comment]
Before unholding this, check that the date is OK. We don't add blog articles with publication dates in the past.

**Comment by divya-mohan0209** [comment]
Are we good to merge this @PurneswarPrasad @sftim ?
The blog looks good to me :)

**Comment by PurneswarPrasad** [comment]
> Are we good to merge this @PurneswarPrasad @sftim ? The blog looks good to me :)

I think the review's completed and we just need to get it out on the mentioned DOP, or earlier if that's possible!

File: content/en/blog/_posts/2022-07-29-SIG-Docs-Spotlight/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+layout: blog
+title: "Spotlight on SIG Docs"
+date: 2022-08-02
+slug: sig-docs-spotlight-2022
+canonicalUrl: https://kubernetes.dev/blog/2022/07/29/sig-docs-spotlight-2022/
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-29-SIG-Docs-Spotlight/index.md:6]:
```suggestion
canonicalUrl: https://kubernetes.dev/blog/2022/08/02/sig-docs-spotlight-2022/
```

**Comment by sftim** [comment]
The path is wrong. I'll fix that in a follow-up PR.

/hold cancel

**Pull request merged**